CRM/Sqltasks/Task.php

```diff
@@ -54,6 +54,11 @@ class CRM_Sqltasks_Task {
   /** @var array generated, sregistered files */
   protected static $files = [];
 
+  /**
+   * @var array Return key pair from ReturnValue Action
+   */
+  protected static $return_values = [];
```

Comment:
I don't think we want to use `static` here. I'm not sure why `$files` is static (it gets cleared on every run in `reset()`, so there doesn't seem to be much of a point), but most properties of this class are instance-level (e.g. `error_count`, `log_messages`). This avoids issues where calling one task from within another task may overwrite return values set by the parent task.